### PR TITLE
Tidies up APC code

### PR DESCRIPTION
**Pull request by mikomyazaki**
These changes were related to my changing the APC UI to a vueui version, however I'm not planning on finishing that right now. So I'd like to just submit this for the moment. UI to come at some point in the future.

Mostly this just replaces magic numbers and makes the APC code actually readable. 

Has no player-facing effects.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!review

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -1247,63 +1194,59 @@
...
+			lighting = autoset(lighting, CHANNEL_OFF)
+			environ = autoset(environ, CHANNEL_OFF)
 			power_alarm.triggerAlarm(loc, src)
-			autoflag = 0
-
-// val 0=off, 1=off(auto) 2=on 3=on(auto)
-// on 0=off, 1=on, 2=autooff
+			autoflag = AUTOFLAG_OFF
 
 obj/machinery/power/apc/proc/autoset(var/val, var/on)
```
Comment (NonQueueingMatt) [inline code/modules/power/apc.dm:1225]:
```suggestion
/obj/machinery/power/apc/proc/autoset(var/val, var/on)
```

File: code/__defines/apc.dm [hunk 2]
```diff
@@ -0,0 +1,62 @@
+//update_state
```
Comment (NonQueueingMatt) [inline code/__defines/apc.dm:1]: feels like these should be defined at the top of the APC.dm and later undefined, but this is also fine tbh

File: code/modules/power/apc.dm (hunk 1 above)
Comment (mikomyazaki) [reply code/modules/power/apc.dm]: Done.

File: code/__defines/apc.dm (hunk 2 above)
Comment (mikomyazaki) [reply code/__defines/apc.dm]: Done.

**Pull request merged**